CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 ## 1.13.3 (Unreleased)
 FEATURES:
+* Add support for readiness_gate on Pod spec (#949)
```

Comment:
#949 is the PR. Usually we link to the issue https://github.com/hashicorp/terraform-provider-kubernetes/issues/811